<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>972</width>
    <height>674</height>
   </rect>
  </property>
  <property name="focusPolicy">
   <enum>Qt::WheelFocus</enum>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="maximumSize">
       <size>
        <width>250</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="tabPosition">
       <enum>QTabWidget::North</enum>
      </property>
      <property name="tabShape">
       <enum>QTabWidget::Rounded</enum>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tabVoxels">
       <attribute name="title">
        <string>Voxels</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <layout class="QFormLayout" name="formLayout">
          <property name="leftMargin">
           <number>5</number>
          </property>
          <property name="topMargin">
           <number>5</number>
          </property>
          <property name="rightMargin">
           <number>5</number>
          </property>
          <property name="bottomMargin">
           <number>5</number>
          </property>
          <item row="0" column="1">
           <widget class="QDoubleSpinBox" name="spinVoxelSize">
            <property name="alignment">
             <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
            </property>
            <property name="minimum">
             <double>0.010000000000000</double>
            </property>
            <property name="singleStep">
             <double>0.100000000000000</double>
            </property>
            <property name="value">
             <double>0.500000000000000</double>
            </property>
           </widget>
          </item>
          <item row="0" column="0">
           <widget class="QLabel" name="label">
            <property name="text">
             <string>voxel size</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>max range</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QDoubleSpinBox" name="spinMaxRange">
            <property name="alignment">
             <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
            </property>
            <property name="value">
             <double>50.000000000000000</double>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QSpinBox" name="spinPriorScans">
            <property name="alignment">
             <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
            </property>
            <property name="minimum">
             <number>0</number>
            </property>
            <property name="value">
             <number>0</number>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="label_4">
            <property name="text">
             <string>#prior scans </string>
            </property>
           </widget>
          </item>
          <item row="3" column="0">
           <widget class="QLabel" name="label_5">
            <property name="text">
             <string>#past scans</string>
            </property>
           </widget>
          </item>
          <item row="3" column="1">
           <widget class="QSpinBox" name="spinPastScans">
            <property name="alignment">
             <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
            </property>
            <property name="minimum">
             <number>1</number>
            </property>
            <property name="value">
             <number>10</number>
            </property>
           </widget>
          </item>
          <item row="4" column="0">
           <widget class="QRadioButton" name="rdoTrainVoxels">
            <property name="text">
             <string>train</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="6" column="0" colspan="2">
           <widget class="QCheckBox" name="chkShowPoints">
            <property name="text">
             <string>show points</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="5" column="0" colspan="2">
           <widget class="QCheckBox" name="chkShowVoxels">
            <property name="text">
             <string>show voxels</string>
            </property>
           </widget>
          </item>
          <item row="7" column="0" colspan="2">
           <widget class="QCheckBox" name="chkShowOccluded">
            <property name="text">
             <string>occluded voxels</string>
            </property>
           </widget>
          </item>
          <item row="8" column="0" colspan="2">
           <widget class="QCheckBox" name="chkShowInvalid">
            <property name="text">
             <string>invalid voxels</string>
            </property>
           </widget>
          </item>
          <item row="4" column="1">
           <widget class="QRadioButton" name="rdoTestVoxels">
            <property name="text">
             <string>test</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QGridLayout" name="gridLayout">
          <property name="leftMargin">
           <number>5</number>
          </property>
          <property name="topMargin">
           <number>5</number>
          </property>
          <property name="rightMargin">
           <number>5</number>
          </property>
          <property name="bottomMargin">
           <number>5</number>
          </property>
          <item row="3" column="0">
           <widget class="QLabel" name="label_7">
            <property name="text">
             <string>y</string>
            </property>
           </widget>
          </item>
          <item row="4" column="0">
           <widget class="QLabel" name="label_8">
            <property name="text">
             <string>z</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QDoubleSpinBox" name="spinMinExtentX">
            <property name="decimals">
             <number>1</number>
            </property>
            <property name="minimum">
             <double>-1000.000000000000000</double>
            </property>
            <property name="maximum">
             <double>1000.000000000000000</double>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QLabel" name="label_9">
            <property name="text">
             <string>min</string>
            </property>
           </widget>
          </item>
          <item row="1" column="2">
           <widget class="QLabel" name="label_10">
            <property name="text">
             <string>max</string>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="label_6">
            <property name="text">
             <string>x</string>
            </property>
           </widget>
          </item>
          <item row="3" column="1">
           <widget class="QDoubleSpinBox" name="spinMinExtentY">
            <property name="decimals">
             <number>1</number>
            </property>
            <property name="minimum">
             <double>-1000.000000000000000</double>
            </property>
            <property name="maximum">
             <double>1000.000000000000000</double>
            </property>
           </widget>
          </item>
          <item row="2" column="2">
           <widget class="QDoubleSpinBox" name="spinMaxExtentX">
            <property name="decimals">
             <number>1</number>
            </property>
            <property name="minimum">
             <double>-1000.000000000000000</double>
            </property>
            <property name="maximum">
             <double>1000.000000000000000</double>
            </property>
           </widget>
          </item>
          <item row="3" column="2">
           <widget class="QDoubleSpinBox" name="spinMaxExtentY">
            <property name="decimals">
             <number>1</number>
            </property>
            <property name="minimum">
             <double>-1000.000000000000000</double>
            </property>
            <property name="maximum">
             <double>1000.000000000000000</double>
            </property>
           </widget>
          </item>
          <item row="4" column="2">
           <widget class="QDoubleSpinBox" name="spinMaxExtentZ">
            <property name="decimals">
             <number>1</number>
            </property>
            <property name="minimum">
             <double>-1000.000000000000000</double>
            </property>
            <property name="maximum">
             <double>1000.000000000000000</double>
            </property>
           </widget>
          </item>
          <item row="4" column="1">
           <widget class="QDoubleSpinBox" name="spinMinExtentZ">
            <property name="decimals">
             <number>1</number>
            </property>
            <property name="minimum">
             <double>-1000.000000000000000</double>
            </property>
            <property name="maximum">
             <double>1000.000000000000000</double>
            </property>
           </widget>
          </item>
          <item row="0" column="0" colspan="3">
           <widget class="QLabel" name="label_11">
            <property name="text">
             <string>Extent</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <spacer name="verticalSpacer_2">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_2">
       <attribute name="title">
        <string>Visuals</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <layout class="QFormLayout" name="formLayout_2">
          <property name="leftMargin">
           <number>5</number>
          </property>
          <property name="topMargin">
           <number>5</number>
          </property>
          <property name="rightMargin">
           <number>5</number>
          </property>
          <property name="bottomMargin">
           <number>5</number>
          </property>
          <item row="0" column="1">
           <widget class="QSpinBox" name="spinPointSize">
            <property name="alignment">
             <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
            </property>
            <property name="minimum">
             <number>1</number>
            </property>
            <property name="maximum">
             <number>20</number>
            </property>
           </widget>
          </item>
          <item row="0" column="0">
           <widget class="QLabel" name="label_3">
            <property name="text">
             <string>point size</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QCheckBox" name="chkShowColor">
          <property name="text">
           <string>color</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="chkShowRemission">
          <property name="text">
           <string>remission</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <property name="leftMargin">
       <number>5</number>
      </property>
      <property name="topMargin">
       <number>5</number>
      </property>
      <property name="rightMargin">
       <number>5</number>
      </property>
      <property name="bottomMargin">
       <number>5</number>
      </property>
      <item>
       <widget class="Viewport" name="mViewportXYZ" native="true">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="MinimumExpanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>640</width>
          <height>480</height>
         </size>
        </property>
        <property name="focusPolicy">
         <enum>Qt::WheelFocus</enum>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_5">
        <property name="leftMargin">
         <number>5</number>
        </property>
        <property name="topMargin">
         <number>5</number>
        </property>
        <property name="rightMargin">
         <number>5</number>
        </property>
        <property name="bottomMargin">
         <number>5</number>
        </property>
        <item>
         <widget class="QToolButton" name="btnBackward">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="minimumSize">
           <size>
            <width>25</width>
            <height>25</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>25</width>
            <height>25</height>
           </size>
          </property>
          <property name="text">
           <string>&lt;</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSlider" name="sldTimeline">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="tickPosition">
           <enum>QSlider::TicksBelow</enum>
          </property>
          <property name="tickInterval">
           <number>100</number>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="btnForward">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="minimumSize">
           <size>
            <width>25</width>
            <height>25</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>25</width>
            <height>25</height>
           </size>
          </property>
          <property name="text">
           <string>&gt;</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionOpen"/>
   <addaction name="actionSave"/>
   <addaction name="separator"/>
   <addaction name="actionRecord"/>
   <addaction name="actionSnapshot"/>
  </widget>
  <action name="actionOpen">
   <property name="text">
    <string>&amp;Open...</string>
   </property>
  </action>
  <action name="actionPaintMode">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Paint</string>
   </property>
  </action>
  <action name="actionCylinderMode">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Cylinder</string>
   </property>
  </action>
  <action name="actionNewCylinder">
   <property name="text">
    <string>NewCylinder</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>&amp;Save...</string>
   </property>
  </action>
  <action name="action_Save">
   <property name="text">
    <string>&amp;Save</string>
   </property>
  </action>
  <action name="action_Labels">
   <property name="text">
    <string>&amp;Labels...</string>
   </property>
  </action>
  <action name="actionTiled">
   <property name="text">
    <string>Tiles</string>
   </property>
  </action>
  <action name="actionRecord">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>record</string>
   </property>
  </action>
  <action name="actionSnapshot">
   <property name="text">
    <string>snapshot</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>Viewport</class>
   <extends>QWidget</extends>
   <header>widget/Viewport.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
